projects/ui/src/lib/components/po-field/po-decimal/po-decimal.component.spec.ts

```diff
@@ -1251,6 +1253,40 @@ describe('PoDecimalComponent:', () => {
       expect(component.hasLetters(undefined)).toBeFalsy();
     });
 
+    it('isGreaterThanTotalLengthLimit: should return `false` if total sum is 16.', () => {
```

Comment:
Seria interessante organizar os testes em describe('isGreaterThanTotalLengthLimit:',()=>{
it('should return `false` if total sum is 16.', () => {
})

Reply:
Organizamos em describe apenas quando a quantidade excede 3 testes...